packages/hooks/rakki/useRakkiTicketsByUser.ts

```diff
@@ -0,0 +1,43 @@
+import { useQuery } from "@tanstack/react-query";
+
+import { RakkiQueryClient } from "../../contracts-clients/rakki/Rakki.client";
+import {
+  NetworkFeature,
+  getNetworkFeature,
+  getNonSigningCosmWasmClient,
+  parseUserId,
+} from "../../networks";
+
+export const useRakkiTicketsCountByUser = (userId?: string) => {
+  const { data: ticketsCount = null, ...other } = useQuery(
+    ["rakkiTicketsCountByUser", userId],
+    async () => {
+      if (!userId) {
```

Comment:
I think we can use useQuery option: enabled: !!userId

Reply:
Yes!
https://github.com/TERITORI/teritori-dapp/pull/1434/commits/a1ada4e8dcf9f6dc572d932ea187ec57608f4c1d